src/main/java/nl/tudelft/pixelperfect/Game.java

```diff
@@ -1,5 +1,7 @@
 package nl.tudelft.pixelperfect;
 
+import java.io.IOException;
+
```

Comment:
Hmm, that is strange indeed... I just cleaned and built the project on two different machines, both reports are clean. Can you try to remove your target directory, clean your project, rebuild and see if the error persists?